core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -213,6 +214,9 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
                           }
       } yield {
         entities
+      }) recover {
+        case dle: DeltaLayerException =>
+          throw new RawlsExceptionWithErrorReport(ErrorReport(dle.code, dle.getMessage, dle))
```

Comment:
without this `recover`, any DeltaLayerExceptions will be obscured and their status codes dropped; all exceptions would result in 500s to the user